Sources/Valet/Internal/Keychain.swift

```diff
@@ -265,73 +265,94 @@ internal final class Keychain {
                 throw error
             }
         }
-        
+
         // Sanity check that we are capable of migrating the data.
-        var keysToMigrate = Set<String>()
+        var keyValuePairsToMigrate = [String: Data]()
         for keychainEntry in retrievedItemsToMigrateWithData {
-            guard let key = keychainEntry[kSecAttrAccount as String] as? String, key != Keychain.canaryKey else {
```

Comment:
We'll force the `key` to be a `String` later on as part of the `compactMap`. For now, we just want to make sure we _have_ a key.